client/ayon_nuke/api/plugin.py

```diff
@@ -452,6 +455,12 @@ def _get_render_target_enum(self):
         )
 
     def create(self, product_name, instance_data, pre_create_data):
+        if not pre_create_data:
+            # add no selection for headless
+            pre_create_data = {
+                "use_selection": False
+            }
```

Comment:
This seems odd to do. Especially because this doesn't necessarily differentiate between "headless" (e.g. Nuke without UI) or running the creator through a script, like e.g. template placeholders.

If the issue is to solve template placeholders, then you may be better off making sure that the Loader used there actually clears the selection?